.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Issue template
```

Comment:
use it only for bug reports? 
If so then maybe a title like "Bug reports" should be better,

Reply:
Done in 7e5deae